marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -105,7 +106,11 @@ public void createProject(String projectDirString) {
         return getHubConfigDir().resolve("security");
     }
 
-    @Override public Path getHubSchemasDir() { return getHubConfigDir().resolve("schemas"); }
+    @Override public Path getHubSchemasDir() { return this.stagingSchemasDir; }
```

Comment:
I would remove this, along with the private stagingSchemasDir variable. I don't think there's any reason for it - other than perhaps backwards compatibility, but I don't think that's necessary in this case because a feature is being removed - i.e. there's no need to keep track of this.

Reply:
@aebadirad , 
Do you want to retain the getHubSchemasDIr() for backward compatibility for 4.2 release?

Reply:
Yes. Mark it deprecated though.